plugins/webp-uploads/settings.php

```diff
@@ -48,45 +77,41 @@ function webp_uploads_add_media_settings_field() {
 add_action( 'admin_init', 'webp_uploads_add_media_settings_field' );
 
 /**
- * Renders the settings field for the 'perflab_generate_webp_and_jpeg' setting.
+ * Renders the settings field for the 'perflab_generate_avif_and_webp' setting.
  *
- * @since 1.0.0
+ * @since n.e.x.t
  */
-function webp_uploads_generate_webp_jpeg_setting_callback() {
-	if ( ! is_multisite() ) {
-		?>
-			</td>
-			<td class="td-full">
-		<?php
-	}
+function webp_uploads_generate_avif_webp_setting_callback() {
+
+	$selected = get_option( 'perflab_generate_avif_and_webp', 'avif' );
 	?>
-		<label for="perflab_generate_webp_and_jpeg">
-			<input name="perflab_generate_webp_and_jpeg" type="checkbox" id="perflab_generate_webp_and_jpeg" aria-describedby="perflab_generate_webp_and_jpeg_description" value="1"<?php checked( '1', get_option( 'perflab_generate_webp_and_jpeg' ) ); ?> />
-			<?php esc_html_e( 'Generate JPEG files in addition to WebP', 'webp-uploads' ); ?>
-		</label>
-		<p class="description" id="perflab_generate_webp_and_jpeg_description"><?php esc_html_e( 'Enabling JPEG in addition to WebP can improve compatibility, but will effectively double the filesystem storage use of your images.', 'webp-uploads' ); ?></p>
+			<label for="perflab_generate_avif_and_webp">
+				<select name="perflab_generate_avif_and_webp" id="perflab_generate_avif_and_webp" aria-describedby="perflab_generate_avif_and_webp_description">
```

Comment:
my assumption is that it only ever makes sense to output to a single format (plus optionally the JPEG fallback) - hence using a dropdown for format selection here